opdracht1.js

```diff
@@ -0,0 +1,24 @@
+// Opdracht 1 - Array Methoden
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+console.log("opdracht 1a");
+const tvTypes = inventory.map((tv) => tv.type);
+console.log({ tvTypes });
+
+//  Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+console.log("opdracht 1b");
+const tvSoldOut = inventory.filter((tv) => tv.sold === tv.originalStock);
+console.log({ tvSoldOut });
+
+// Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+console.log("opdracht 1c");
+const tvAmbiLight = inventory.filter((tv) => {
+  return tv.options.ambiLight === true;
```

Comment:
Goed hoor, maar kan nog ietjes korter 😉 
```suggestion
  return tv.options.ambiLight;
```